README.md

```diff
@@ -0,0 +1,38 @@
+# SpamWatch API
+This repository is where the source code lies for the SpamWatch API, written in Rust.
+
+**NOTE: The SpamWatch API is in beta phrase, but it successfully worked in production.**
+
+## Setup
+### What You Need?
+- Rust
+- Postgres for database
+- Web server with public-reachable address like `https://api.spamwat.ch`.
+
+### Manual Builds
+1. Get your copy of this code with `git clone https://github.com/SPamWatch/SpamWatchAPI.git`.
+2. Install dependencies with `cargo build`.
```

Comment:
This downloads and compiles dependencies and then compiles a debug build which should only be used for local development